src/app/features/bird-info/models.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  DataGridFilter,
+  DataGridFiltersObj,
+  GridDataResponse,
+  GridQuery
+} from "../../../components/table/DataGridModels";
+import { UserInfo } from "../auth/models";
+
+export enum VerificationStatus {
+  Pending = "Pending",
+  Verified = "Verified",
+  Rejected = "Rejected"
+}
+
+export interface BirdObservationData {
```

Comment:
these types are better be reused from observation list models too, to avoid duplication

Reply:
fixed